test/integration/test_contact_mechanics.py

```diff
@@ -165,6 +166,19 @@ def bc_type(self, g):
         bc.is_dir[:, frac_face] = True
         return bc
 
+    def set_parameters(self):
+        super().set_parameters()
+        dilation_angle = getattr(self, "dilation_angle", 0)
+        for g, d in self.gb:
+            if g.dim < self.Nd:
+
+                initial_gap = getattr(self, "initial_gap", np.zeros(g.num_cells))
+
+                d[pp.PARAMETERS]["mechanics"].update(
+                    {"initial_gap": initial_gap, "dilation_angle": dilation_angle}
+                )
+
 
 if __name__ == "__main__":
-    unittest.main()
+    TestContactMechanics().test_displace_south_x_nonconstant_gap()
```

Comment:
```suggestion
    
```

Reply:
I think we don't want the blank line, right?
```suggestion
```